packages/template-retail-react-app/config/default.js

```diff
@@ -57,7 +57,7 @@ module.exports = {
                 path: 'api'
             },
             {
-                host: 'zzrf-001.dx.commercecloud.salesforce.com',
+                host: 'zzrf-002.dx.commercecloud.salesforce.com',
```

Comment:
I don't think we want to commit any of these config changes since it is being applied to our general template-retail-react-app and not just the e2e tests